2_observations.yml

```diff
@@ -28,10 +36,31 @@ targets:
     
   out_data/reservoir_releases.csv:
     command: file.copy(from = '../delaware-model-prep/2_observations/out/reservoir_releases.csv', to = target_name)
-  
+
   out_data/reservoir_releases_lordville.csv:
```

Comment:
maybe keep consistent naming conventions when it is a reservoir file? I think now we're going between "reservoir" and "res"